main.go

```diff
@@ -119,10 +123,14 @@ func displayVersion() {
 	fmt.Printf("%s\n", detectRTInfo().version())
 }
 
-func fail(err error) {
+func emit(err error) {
```

Comment:
nit: maybe `emitError` as it write on stderr

Reply:
Hmmm. Yeah, I think that would be better.